frontend/public/components/replication-controller.jsx

```diff
@@ -27,14 +26,15 @@ import {
 } from './utils';
 
 import { VolumesTable } from './volumes-table';
-import { confirmModal } from './modals';
+import { confirmModal, rollBackRCModal } from './modals';
 import { k8sPatch } from '../module/k8s';
 
+const INACTIVE_STATUSES = ['New', 'Pending', 'Running'];
+
 const CancelAction = (kind, obj) => ({
   // t('public~Cancel rollout')
   labelKey: 'public~Cancel rollout',
-  hidden: !_.includes(
-    ['New', 'Pending', 'Running'],
+  hidden: !INACTIVE_STATUSES.includes(
     obj?.metadata?.annotations?.['openshift.io/deployment.phase'],
```

Comment:
We should make sure this action is hidden if the annotation is missing or the replication controller isn't owned by a deployment config.

Reply:
@spadgett Do you want this change for `Cancel Rollout` or for `Rollback` (or both)?

Reply:
@jeff-phillips-18 You can use the owner references to see if it's owned by a deployment config or a standalone replication controller.

The Rollback action might be OK since we're already checking the deployment phase annotation (which should only be there if it's owned), but it's probably better to check both places.

Reply:
@spadgett I've added the check for `deployment.phase` annotation to both the cancel and rollback actions.